aea/protocols/base.py

```diff
@@ -230,12 +232,14 @@ def __str__(self):
             "Message(sender={},to={},".format(self._sender, self._to)
             + ",".join(
                 map(
-                    lambda key_value: str(key_value[0]) + "=" + str(key_value[1]),
+                    lambda key_value: str(key_value[0])[:MAX_PRINT_INNER]
+                    + "="
+                    + str(key_value[1])[:MAX_PRINT_INNER],
                     self._body.items(),
                 )
             )
             + ")"
-        )
+        )[:MAX_PRINT_OUTER]
```

Comment:
@solarw , @marcofavorito added to prevent log spamming. Do you have any concerns?

Reply:
looks good to me